tests/integration/workflows/test_workflows_tasks_upload.py

```diff
@@ -26,14 +26,24 @@
 from flask import current_app
 
 from invenio_workflows import workflow_object_class
-from invenio_workflows.models import Workflow
 
 
 # FIXME: otherwise this task is not found by Celery.
 from inspirehep.modules.orcid.tasks import orcid_push  # noqa: F401
 from inspirehep.modules.workflows.tasks.upload import store_record
 
 
+class AttrDict(dict):
+    def __init__(self, *args, **kwargs):
+        super(AttrDict, self).__init__(*args, **kwargs)
+        self.__dict__ = self
```

Comment:
This looks very ugly to me :/, as it will have the actual methods of the dict (for example, update).

Reply:
I would use another mock object, like `MockEng` or similar (even a MagickMock from the mock module).

Reply:
This is exactly how we do it on the unit tests https://github.com/inspirehep/inspire-next/blob/master/tests/unit/helpers/mocks.py#L30-L33